src/index.html

```diff
@@ -63,20 +63,64 @@ <h5 class="text-light h5 font-weight-bold my-4">Teaching girls about technology
       </section>
 
       <section class="col-12 justify-content-center bg-grey">
-        <p id="description" class="text-center mx-auto">
+        <p id="description" class="text-center col-xs-8 col-md-6 col-lg-4 mx-auto my-5">
           Girls Code Lincoln was established in January 2016 as Lincoln Girls Who Code. We host technology clubs
           in Lincoln for 4th - 9th grade girls. Since inception we have held over 15 clubs and impacted over 200
           girls.
         </p>
       </section>
 
-      <section id="calendar" class="bg-pink col-12">
-        <iframe src="https://calendar.google.com/calendar/embed?mode=AGENDA&amp;height=400&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=eh2akbgitkihie7k4svqg256u8%40group.calendar.google.com&amp;color=%235229A3&amp;ctz=America%2FChicago"
-          style="border-width:0" width="100%" height="400" frameborder="0" scrolling="no"></iframe>
+      <section id="past-clubs" class="bg-pink col-12 py-5">
+        <h2 class="display3 text-center mb-5">Our Past Code Clubs</h2>
+        <div class="col-12 d-flex justify-content-center">
+          <div class="col-lg-6 col-xs-8">
+            <ul class="timeline">
+              <li>
```

Comment:
Let's order the timeline reverse-chronologically so the recent clubs are at the top